build/deps/tigris.Dockerfile

```diff
@@ -1 +1 @@
-FROM tigrisdata/tigris-local:1.0.0-alpha.26
+FROM tigrisdata/tigris-local:1.0.0-alpha.22
```

Comment:
Please move it back to `alpha.26` as `alpha.22` has some bugs (e.g. we caught one with `delete`).